packages/rehype-mathjax/package.json

```diff
@@ -25,9 +25,9 @@
     "url": "https://opencollective.com/unified"
   },
   "browser": {
-    "./renderer/node": "./renderer/browser"
+    "./renderer/base/node": "./renderer/base/browser"
   },
```

Comment:
`renderer/base/node` is no longer available, in this PR, no? So then this `browser` field can be removed?

Reply:
Seems travis crashes on this too: that there is no `/node` target

Reply:
Excuse me. I forgot to add them.